person.rb

```diff
@@ -1,10 +1,14 @@
+require_relative 'corrector'
+require_relative 'rental'
+
 class Person
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = [1..100]
     @name = name
     @age = age
     @parent_permission = parent_permission
     @corrector = Corrector.new
+    @rentals = []
```

Comment:
- [ ] Getter and Setter for @entals should be present that includes `:name, :age, :rentals`.

Reply:
Done